azext_iot/central/_help.py

```diff
@@ -85,7 +85,7 @@ def _load_central_devices_help():
         examples:
         - name: Get a device
           text: >
-            az iot central app device list
+            az iot central app device delete
```

Comment:
There is an existing command `az iot central device-twin`. Clearly that should be changed to `az iot central app device-twin`?

Reply:
I can work with Jack on this. Whenever iotcentral gets officially deprecated we could just remove all references to `device-twin` and go with the `device` nomenclature since it conforms with the public API's

Reply:
Are you saying for central here, `device-twin` and `device` are equivalent concepts? For IoT Hub device-identity and device-twin (similar for modules) are different entities.

Reply:
Just want to make sure I conveyed this part - the `device-twin show` command also exists in the non-deprecated `iot central`.

Reply:
Oh i see you are correct, yes that should move to `iot central app`. I'll work with Johnson and Jack on this tomorrow

Reply:
but that might need a whole 3 month deprecation cycle